config/settings.yml

```diff
@@ -81,4 +81,6 @@ discourse_subscriptions:
     default: ""
     hidden: true
   discourse_subscriptions_enable_automatic_tax:
+    default: false
+  discourse_subscriptions_enable_verbose_logging:
```

Comment:
Should this be a hidden setting?

Reply:
maybe? I don't know? I want admins to be able to turn it on and be able to look at the logs. But I don't think we need to make them use the rails console to do so.

Reply:
Fair enough, if we want admins to be able to turn it on this is fine. Sometimes we add these kind of debug settings just for ourselves, in that case they are better hidden :+1: